tests/tracer/test_encoders.py

```diff
@@ -110,7 +110,7 @@ def decode(data):
 
 class RefMsgpackEncoderV03(RefMsgpackEncoder):
     def normalize(self, span):
-        d = span.to_dict()
+        d = RefMsgpackEncoderV03._span_to_dict(span)
```

Comment:
Agree this is weird and if we want the encoder API to be public we should rethink this.

For instance, we need to transform a span (with its slots) into a dict for the json encoders or the fallback msgpack `encode` test. However, we don't need this in the custom msgpack encoder.

Better to make this an internal helper rather than a static method on the base encoder?

Reply:
oh, nvm, I think I cannot read, this is just `self._span_to_dict`, which is fine.